packages/dns-forward/dns-forward.0.7.2/opam

```diff
@@ -19,6 +19,7 @@ depends: [
   "topkg"      {build}
   "ppx_tools"  {build}
   "ppx_cstruct" {build}
+  "ppx_sexp_conv" {build}
```

Comment:
i think this also needs an `lwt <3.0.0` constraint

```
#=== ERROR while compiling dns-forward-lwt-unix.0.9.0 =========================#
# context              2.0.0~beta3 | linux/x86_64 | ocaml-base-compiler.4.04.2 | file:///home/opam/src
# path                 ~/.opam/ocaml-base-compiler.4.04.2/.opam-switch/build/dns-forward-lwt-unix.0.9.0
# command              jbuilder build -p dns-forward-lwt-unix -j 1
# exit-code            1
# env-file             ~/.opam/log/dns-forward-lwt-unix-84-0059a6.env
# output-file          ~/.opam/log/dns-forward-lwt-unix-84-0059a6.out
### output ###
#     ocamldep lwt/dns_forward_lwt_unix.depends.ocamldep-output
#     ocamldep lwt/dns_forward_lwt_unix.dependsi.ocamldep-output
#       ocamlc lwt/dns_forward_lwt_unix.{cmi,cmti}
#       ocamlc lwt/dns_forward_lwt_unix.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/ocaml-base-compiler.4.04.2/bin/ocamlc.opt -w -40 -g -bin-annot -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/astring -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/base64 -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/bytes -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/cstruct -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/cstruct-lwt -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/dns -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/dns-forward -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/duration -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/fmt -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/hashcons -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/io-page -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/io-page-unix -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/ipaddr -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/logs -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/lwt -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/mirage-channel -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/mirage-channel-lwt -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/mirage-clock -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/mirage-clock-lwt -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/mirage-clock-unix -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/mirage-device -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/mirage-flow -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/mirage-flow-lwt -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/mirage-time -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/mirage-time-lwt -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/ocaml -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/ocplib-endian -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/re -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/result -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/rresult -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/sexplib -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/sexplib/0 -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/stringext -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/uchar -I /home/opam/.opam/ocaml-base-compiler.4.04.2/lib/uri -no-alias-deps -I lwt -o lwt/dns_forward_lwt_unix.cmo -c -impl lwt/dns_forward_lwt_unix.ml)
# File "lwt/dns_forward_lwt_unix.ml", line 214, characters 9-22:
# Warning 3: deprecated: Lwt_unix.bind
#  This function will soon evaluate to a promise (-> unit Lwt.t), because the
#  bind system call can block for Unix domain sockets. See
#    https://github.com/ocsigen/lwt/issues/230
#  This will be a breaking change in Lwt 3.0.0.
#  If you don't use Unix domain sockets and use Lwt_unix.bind ... ; rather than
#  let () = Lwt_unix.bind ... in, you can ignore this warning.
#  To retain the current signature, use Lwt_unix.Versioned.bind_1
#  To use the new non-blocking version immediately, use Lwt_unix.Versioned.bind_2
#  Both alternatives require Lwt >= 2.7.0.
# File "lwt/dns_forward_lwt_unix.ml", line 214, characters 9-55:
# Error: This expression has type unit but an expression was expected of type
#          'a Lwt.t

```